src/components/tasks/card/index.tsx

```diff
@@ -117,6 +141,16 @@ const Card: FC<Props> = ({
           </span>
         </span>
       </div>
+      <div className={classNames.cardItems}>
+        <span className={classNames.progressContainer}>
+          <div className={classNames.progressIndicator}>
+            <div className={handleProgressColor(content.percentCompleted, content.startedOn, content.endsOn)} style={{ width: `${content.percentCompleted}%` }}></div>
```

Comment:
We could have multiple classes appended that should resolve in reducing repeated css

Reply:
thanks for this suggestion will do this